test/libsolidity/syntaxTests/nameAndTypeResolution/372_shift_constant_right_fractional.sol

```diff
@@ -2,4 +2,5 @@ contract C {
     uint public a = 0x42 >> (1 / 2);
 }
 // ----
+// TypeError: (33-48): Operator >> not compatible with types int_const 66 and rational_const 1 / 2. Right argument of shift operation should be unsigned.
```

Comment:
Not sure if the error message here make sense. It's repeated and the "should be unsigned" part is misleading.